# Parameters for the simulation
sim:
  propogate_rate: 200.0   # How often the dynamics are updated in Hz
  alpha: 0.0              # Amount of uncertainty (0.1 -> 10% uncertainty)
  model_type: simple      # The type of dynamic model that is used
                          # options include:
                          #   simple - A simple dynamic model of a quadcopter
                          # Add another model type as a child class of EquationsOfMotion
  forces_model_type: flat_momentum
  sensor_type: simple
  wind:
    wind_type: dryden     # Which wind model to use
    wind_rate: 20         # Rate at which the wind model is updated (Hz)
    w_ns: 0.0             # Steady wind speed in north direction (m/s)
    w_es: 0.0             # Steady wind speed in east direction  (m/s)
    w_ds: 0.0             # Steady wind speed in down direction  (m/s)
